src/app-layout/classic.tsx

```diff
@@ -500,20 +499,16 @@ const ClassicAppLayout = React.forwardRef(
                     !contentHeader &&
                     styles['content-wrapper-first-child']
                 )}
+                style={getStickyOffsetVars(
+                  placement.insetBlockStart,
+                  placement.insetBlockEnd + (splitPanelBottomOffset || 0),
+                  `${stickyNotificationsHeight}px`,
+                  mobileBarHeight && !disableBodyScroll ? `${mobileBarHeight}px` : '0px',
+                  !!disableBodyScroll,
+                  isMobile
+                )}
               >
-                <AppLayoutContext.Provider
-                  value={{
-                    stickyOffsetTop:
-                      // We don't support the table header being sticky in case the deprecated disableBodyScroll is enabled,
```

Comment:
Fun fact. Now we do. It was easier to support this feature than to port hacks into the new code